client/src/modules/prices/prices.html

```diff
@@ -31,6 +31,30 @@
                 <i class="fa fa-close"></i> <span translate>FORM.BUTTONS.CLEAR_GRID_CONFIGURATION</span>
               </a>
             </li>
+
+            
+            <li role="menuitem">
+              <a href data-method="clear-state" ng-click="PriceListCtrl.clearGridState()">
+                <i class="fa fa-close"></i> <span translate>FORM.BUTTONS.CLEAR_GRID_CONFIGURATION</span>
```

Comment:
It looks like you repeated this option?   Isn't it the same button as on line 31?

Reply:
you're right! let me remove it then.